libretroshare/src/file_sharing/hash_cache.cc

```diff
@@ -117,8 +117,8 @@ void HashStorage::data_tick()
                 if(!mChanged)	// otherwise it might prevent from saving the hash cache
                 {
                     std::cerr << "Stopping hashing thread." << std::endl;
-                    shutdown();
-                    mRunning = false ;
+					askForStop();
```

Comment:
@G10h4ck is just following the coding style. Everybody else should do the same!
https://github.com/RetroShare/RetroShare/wiki/Coding#coding-style